app/assets/stylesheets/components/modals.scss

```diff
@@ -43,6 +43,10 @@
     top: 0;
     bottom: 0;
     pointer-events: auto;
+
+    &.clickable {
+      pointer-events: none;
+    }
```

Comment:
This is to make sure the click event passes through to `focus-trap` which then can handle the 'click outside' behaviour